### ISSUE TITLE
Link to repository missing on pyfilesystem.org

### ISSUE THREAD

**Issue by hmeine** [PyFilesystem/pyfilesystem#225]
On http://pyfilesystem.org, I could not find a link to this GitHub repository.  (I know, this is not really an issue with the code in this repository, but I wanted to post this issue and have it tracked.)

**Comment by arekbulski** [comment]
Good point. Indeed, the main page refers to neither hosting site. Docs section, Introduction page refers to Google Code, and Bugs section leads to Google Code issues page directly.

**Comment by lurch** [comment]
#235 fixes the references to Google Code, but @willmcgugan needs to update http://pyfilesystem.org/ and http://docs.pyfilesystem.org/

**Comment by willmcgugan** [comment]
Thanks, Andrew. Just updated put the git repos in readthedocs. It should update shortly.

**Comment by lurch** [comment]
@willmcgugan http://docs.pyfilesystem.org/en/latest/introduction.html still links to the GoogleCode issues URL?

Also, I just noticed that http://docs.pyfilesystem.org/en/latest/ftpfs.html and http://docs.pyfilesystem.org/en/latest/tempfs.html look okay, but http://docs.pyfilesystem.org/en/latest/sftpfs.html (and http://docs.pyfilesystem.org/en/latest/expose/sftp.html ) is totally blank? Should I submit that as a separate issue? (I dunno anything about docs-generation)

**Comment by lurch** [comment]
Woohoo, I added the ReadTheDocs webhook to this GitHub project, removed `wx` from `requirements.txt`, and now http://docs.pyfilesystem.org/ is up to date again! (The introduction page now links to the github issues list rather than the googlecode issues list)

And the pages I mentioned in my previous comment about being totally blank, now have content :-)

But the links on http://pyfilesystem.org/ itself (which is what this issue was originally opened for) are still out of date...

**Comment by lurch** [comment]
...although something on ReadTheDocs has now gone a bit strange?! https://github.com/rtfd/readthedocs.org/issues/1971#issuecomment-180154869

**Comment by lurch** [comment]
Well ReadTheDocs seems to be working again, so http://docs.pyfilesystem.org is now up to date, but http://pyfilesystem.org/ itself could still do with an update... ping @willmcgugan again ;-)

**Comment by arekbulski** [comment]
Lurch, you mean google groups link could be complemented or replaced by GitHub issues? 

This is stale issue, maybe just close it?

**Comment by lurch** [comment]
No, I mean that http://pyfilesystem.org/ (which is something that only @willmcgugan controls) still links to https://code.google.com/p/pyfilesystem/issues/list instead of https://github.com/PyFilesystem/pyfilesystem/issues , and links to https://pypi.python.org/pypi/fs/0.5.0 instead of https://pypi.python.org/pypi/fs/0.5.4
